webpack/webpack.config.js

```diff
@@ -8,7 +8,29 @@ function requireAll(array) {
   return array.map(require.resolve);
 }
 
+const devServer = {
+  open: false,
+  overlay: true,
+  port: 4200,
+  proxy: require("./proxy.dev.js")
+};
+
+if (process.env.NODE_ENV === "testing") {
+  // Cypress constantly saves fixture files, which causes webpack to detect
+  // a filechange and rebuild the application. The problem with this is that
+  // when Cypress goes to load the application again, it may not be ready to
+  // be served, which causes the test to fail. This delays rebuilding the
+  // application for a very long time when in a testing environment.
+  const delayTime = 1000 * 60 * 60 * 5;
+  devServer.watchOptions = {
```

Comment:
what's the benefit over just [turning watch off](https://webpack.js.org/configuration/watch/) entirely?

Reply:
We could, I just restored the config we've had. I believe we still would like to recompile the code as we adapt it. Completely disabling it would mean stoping and restarting the `npm run start:testing`